Dockerfile

```diff
@@ -19,6 +19,8 @@ RUN apt-get update \
         libunwind8 \
         lsb-release \
         netcat \
+        nodejs \
```

Comment:
note that it is Dockerfile best practice to pin package versions as well

Reply:
Oh, ok. All of them?

Reply:
I've pinned NodeJS and NPM for now.